.github/workflows/check_urls.yml

```
@@ -0,0 +1,46 @@
+name: Check URLs
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: urls-checker-code
+      uses: urlstechie/urlchecker-action@master
```

```suggestion
      uses: urlstechie/urlchecker-action@xxxx
```

For security I would pin a hash

---

done